Directory.Build.props

```diff
@@ -80,20 +80,20 @@
 
   <!-- The TFMs to build and test against. -->
   <PropertyGroup>
-    <NetCoreAppCurrentVersion>8.0</NetCoreAppCurrentVersion>
+    <NetCoreAppCurrentVersion>9.0</NetCoreAppCurrentVersion>
     <NetCoreAppCurrentIdentifier>.NETCoreApp</NetCoreAppCurrentIdentifier>
     <NetCoreAppCurrentTargetFrameworkMoniker>$(NetCoreAppCurrentIdentifier),Version=v$(NetCoreAppCurrentVersion)</NetCoreAppCurrentTargetFrameworkMoniker>
     <MicrosoftNetCoreAppFrameworkName>Microsoft.NETCore.App</MicrosoftNetCoreAppFrameworkName>
     <NetCoreAppCurrentBrandName>.NET $(NetCoreAppCurrentVersion)</NetCoreAppCurrentBrandName>
     <NetCoreAppCurrent>net$(NetCoreAppCurrentVersion)</NetCoreAppCurrent>
 
     <!-- The previous supported .NET version. -->
-    <NetCoreAppPreviousVersion>7.0</NetCoreAppPreviousVersion>
+    <NetCoreAppPreviousVersion>8.0</NetCoreAppPreviousVersion>
     <NetCoreAppPrevious>net$(NetCoreAppPreviousVersion)</NetCoreAppPrevious>
     <NetCoreAppPrevious Condition="'$(DotNetBuildFromSource)' == 'true'">$(NetCoreAppCurrent)</NetCoreAppPrevious>
 
     <!-- The minimum supported .NET version. -->
-    <NetCoreAppMinimum>net6.0</NetCoreAppMinimum>
+    <NetCoreAppMinimum>net8.0</NetCoreAppMinimum>
```

Comment:
We still support net6.0 for another year - are we really ready to move this?  Also I expect we will have projects that expect Previous and Minimum to be different.  @ViktorHofer

Reply:
Doesn't .NET 6.0 go out of support on the day that .NET 9 comes out?

Reply:
We did the same when we switched from .NET 6 to .NET 7: We removed `netcoreapp3.1` and `net5.0` support as when .NET 7 released, those runtimes were already officially out-of-support.

That model worked well with .NET 7 and I would prefer to drop the TFMs now than in the middle of .NET 9 development.

> Also I expect we will have projects that expect Previous and Minimum to be different.

We should fix such wrong assumptions. I don't recall any managed project that requires those two properties to differ. Those were designed to be equal every odd major release version. And, source build already sets all three properties to the same value.